src/AutoBuild.jl

```diff
@@ -1347,7 +1347,7 @@ function build_jll_package(src_name::String,
         end
         println(io)
         println(io)
-        println(io,"For more details about JLL packages and how to use them, see `BinaryBuilder.jl` [documentation](https://juliapackaging.github.io/BinaryBuilder.jl/dev/jll/).")
+        println(io,"For more details about JLL packages and how to use them, see `BinaryBuilder.jl` [documentation](https://docs.binarybuilder.org/dev/jll/).")
```

Comment:
Also this one should be `stable` now
```suggestion
        println(io,"For more details about JLL packages and how to use them, see `BinaryBuilder.jl` [documentation](https://docs.binarybuilder.org/stable/jll/).")
```

Reply:
Done. Also fixed another absolute (instead of relative, using `@ref`) URL inside the manual.